### PR TITLE
Make anchor inputs more prominent

### PR DESCRIPTION
**Pull request by meodai**
https://user-images.githubusercontent.com/608386/192300105-621800c3-8114-43f0-86cc-969a9fffa27e.mov

### CODE REVIEW

**Comment by meodai** [comment]
Question: should it stay "fancy" once the anchor is in the field? `:focus` + `:not(:empty)`?

File: src/frontend/src/flows/displayUserNumber.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ const pageContent = (userNumber: bigint) => html`
     })}
     <h2 class="t-title">Identity Anchor:</h2>
 
-    <div class="c-input c-input--vip c-input--readonly t-vip" data-usernumber>
+    <div class="c-input c-input--vip c-input--readonly t-vip c" data-usernumber>
```
Comment (nmattia) [inline src/frontend/src/flows/displayUserNumber.ts:17]:
```suggestion
    <div class="c-input c-input--vip c-input--readonly t-vip" data-usernumber>
```

typo?

**Review by nmattia — COMMENTED** [review]
Looking really really nice!

One question: the halo stays the same colour even on bad key press, whereas before it'd turn blue-ish (bruised). Is that on purpose?

Also I've noticed that a solid white rectangle appears behind the input on bad input:

https://user-images.githubusercontent.com/6930756/192302443-e41064c8-4abc-45ad-ad34-acc5859b63e8.mov

**Review by nmattia — APPROVED** [review]
Looks great, thanks!

**Pull request merged**